internal/xds/bootstrap/tlscreds/bundle_ext_test.go

```diff
@@ -224,6 +228,87 @@ func (s) TestCaReloading(t *testing.T) {
 	}
 }
 
+func (s) TestSPIFFEReloading(t *testing.T) {
```

Comment:
Optional: Given how many upper case letters there are in this test name, you could consider using some underscores. We do that *only* for test names to make them more readable (sometimes test names can get really long).

Reply:
Done, especially when we combine MTLS and SPIFFE I think it makes it a lot easier